solidity/contracts/bridge/Bridge.sol

```diff
@@ -1460,68 +1474,38 @@ contract Bridge is Ownable {
-    //            `minimalAmount` in TBTC. Reduce the `pendingRedemptionsValue`
-    //            (`wallets` mapping) for given wallet accordingly.
-    //          - If there are outputs not corresponding to any request - put
-    //            them to the `invalidRedemptions` mapping.
-    //          - If the given output is a change it should be treated
-    //            differently, depending on how other outputs looks like.
-    //            All transactions transferring funds back to wallet PKH
-    //            without doing anything else, burns the main UTXO for fees.
-    //            Wallet should be punished for that and the main UTXO key
-    //            should be put to `invalidRedemptions` as usual.
-    //          - If there are outputs with zero value - put them to the
-    //            `invalidRedemptions` mapping. It doesn't matter if they
-    //            are provably unspendable zero-value false return (OP_RETURN)
-    //            or non-false return outputs, they can be used to burn main
-    //            UTXO value via fees so transaction that uses them should be
-    //            considered fraudulent.
-    //
-    //          There is no need to revert if given redemption key is already
-    //          in `invalidRedemptions` mapping. However, one must count the
-    //          number of unique non-overwriting inserts under `invalidCount`.
-    //          This is because there are some edge cases here:
-    //          - A redemption key looking as fraudulent due to being non-expected,
-    //            can be already present in `invalidRedemptions` due to being a
-    //            timeout. Normally we should not deem it as a new invalid
-    //            redemption and count it to `invalidCount` in order to avoid
-    //            punishing the wallet twice for the same timeout. In the same
-    //            time, it can be an actual fraud leveraging the fact the given
-    //            redemption key was already punished for timeout. To detect
-    //            such a case we can store the timed out request amount as
-    //            `invalidRedemptions` mapping value, and punish for fraud in
-    //            case the transferred value exceed the redeemable amount from
-    //            the timed out request.
-    //          - A fraudulent transaction can deliberately use multiple
-    //            outputs with same script hash. This will produce the same
-    //            redemption key which must be put to `invalidRedemptions`
-    //            multiple times so it should not cause a revert and it is
-    //            enough to count it once in `invalidRedemptions`.
-    //       6. If `invalidCount > 0` transaction is fraudulent. If not - revert.
-    //       7. Punish the wallet, probably by slashing its operators.
-    //       8. Change wallet's state in `wallets` mapping to `MovingFunds` in
+    //       3. Check if wallet state is Active or MovingFunds
```

Comment:
If not, revert.

Reply:
See https://github.com/keep-network/tbtc-v2/pull/128/commits/64395ffafef55a2795e927a1dad3c531d0ef24d8